### PR TITLE
Apple.iPodReset version 1.0.4.71

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98686)

### CODE REVIEW

**Comment by stephengillie** [comment]
I'm not entirely sure, but based on one of the log messages I'm guessing Microsoft.DotNet.Framework.DeveloperPack_4 might be a dependency here as well.

File: manifests/a/Apple/iPodReset/1.0.4.71/Apple.iPodReset.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Apple.iPodReset
+PackageVersion: 1.0.4.71
+PackageLocale: en-US
+Publisher: Apple Inc.
+PackageName: iPod Reset Utility
+License: Propreitary
+Copyright: © 2008 Apple Inc. All Rights Reserved.
+ShortDescription: iPod Reset Utility Installer
```
Comment (Trenly) [inline manifests/a/Apple/iPodReset/1.0.4.71/Apple.iPodReset.locale.en-US.yaml:11]: Please provide a better description of what the utility does

File: manifests/a/Apple/iPodReset/1.0.4.71/Apple.iPodReset.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Apple.iPodReset
+PackageVersion: 1.0.4.71
+Installers:
+- InstallerUrl: https://download.info.apple.com/Mac_OS_X/061-4529.20080508.PdLde/iPodResetUtilitySetup.exe
+  Architecture: neutral
```
Comment (Trenly) [inline manifests/a/Apple/iPodReset/1.0.4.71/Apple.iPodReset.installer.yaml:8]:
```suggestion
  Architecture: x86
  UnsupportedOSArchitectures:
    - x64
    - arm64
```
> Action start 9:11:08: LaunchConditions.
MSI (s) (7C:A8) [09:11:08:990]: Product: iPod Reset Utility -- This installer is intended for 32-bit versions of Windows.

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
Close with reason: Stale PR;

**Comment by upintheairsheep** [comment]
> Close with reason: Stale PR;

Reopen please

**Comment by upintheairsheep** [comment]
@Trenly may you please re-open this PR?

**Comment by Trenly** [comment]
> @Trenly may you please re-open this PR?

There is no way for moderators to reopen PRs

**Pull request closed without merging**